flink-libraries/flink-table/src/main/java/org/apache/flink/table/calcite/rules/FlinkAggregateExpandDistinctAggregatesRule.java

```diff
@@ -114,15 +114,21 @@ public FlinkAggregateExpandDistinctAggregatesRule(
 		this.useGroupingSets = useGroupingSets;
 	}
 
-	@Deprecated // to be removed before 2.0
+	/**
+	 * @deprecated to be removed before 2.0
```

Comment:
These are actually deprecation annotations by Calcite. We copied the files (see comment on top) because we had to fix a bug. The files will be removed once the bugs are fixed with a newer Calcite release.